nmea0183: Add parsing of Quectel PQTMGPS message

Qbicz opened the pull request.

Qbicz commented:
Ready to review

KaSroka commented on drivers/driver_nmea0183.c:
```diff
@@ -3848,6 +3845,46 @@ static gps_mask_t processPQTMIMU(int count, char *field[],
+        "PQTMGPS: time=%d, lat=%.9f lon=%.9f",
+        atoi(field[1]),
+        session->newdata.latitude,
+        session->newdata.longitude
```
I think you should log `wanglem` and `wangler` here.

Qbicz replied:
Fixed

KaSroka reviewed (approved):
Looks good, just one small comment.

Qbicz committed:
nmea0183: Do not process velocity in PQTMINS

This is not needed in the output data.

Qbicz committed:
nmea0183: Add parsing of Quectel PQTMGPS message

The latitude and longitude from GNSS-only position (not including DR
algorithm) are put in the wind angle variables, as a workaround to not
have to add new variables to gpsd gps_fix_t struct.

9 decimal places for these variables are put into the JSON, to
accomodate the lat/lon accuracy.
